angular/angular.json

```diff
@@ -37,7 +37,7 @@
           }
         },
         "serve": {
-          "builder": "@@angular-devkit/build-angular:dev-server",
```

Comment:
I think this is the fix right?

Reply:
yes along with that added the build package